sbin/common-functions.sh

```diff
@@ -49,6 +50,22 @@ checkingAndDownloadingAlsa()
   fi
 }
 
+# Freemarker for OpenJ9
+checkingAndDownloadingFreemarker()
+{
+  echo "Checking for FREEMARKER"
+
+  FOUND_FREEMARKER=$(find "${WORKING_DIR}" -name "freemarker-${FREEMARKER_LIB_VERSION}")
```

Comment:
_Minor comment:_  consider adding `-type d`  to the find criteria?

Reply:
yep, will do. thanks!